ui/packages/app/src/components/Checkbox/Checkbox.svelte

```diff
@@ -4,7 +4,7 @@
 	export let value: boolean = false;
 	export let default_value: boolean = false;
 	export let style: string = "";
-	export let label: string;
```

Comment:
Question: How does the label is realized when the user provides one?

Reply:
@pngwn the default label is assigned like this:
`export let label: string = "Checkbox";`

I was wondering how do we assign the label given by the user

Reply:
These `export let` statement essentially marks the interface for the component. They are passed in from above : `<Checkbox label="some_label" />`. Same way arguments are marked on a function and passed in by the caller.